packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -28,14 +30,26 @@ export default class VTable {
      */
     col: number;
 
+    /**
+     * Selected range of cells with the coordinates of the first and last cell selected.
+     */
+    selection: TableSelection;
+
     private trs: HTMLTableRowElement[] = [];
 
     /**
      * Create a new instance of VTable object using HTML TABLE or TD node
      * @param node The HTML Table or TD node
+     * @param normalizeSize Whether table size needs to be normalized
      */
     constructor(node: HTMLTableElement | HTMLTableCellElement, normalizeSize?: boolean) {
         this.table = safeInstanceOf(node, 'HTMLTableElement') ? node : getTableFromTd(node);
+        this.selection = {

```

Comment:
How about just set the whole selection object to null when there is no table selection?
What do you think?